airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/TrackingMetadata.java

```diff
@@ -35,7 +35,8 @@ public static ImmutableMap<String, Object> generateSyncMetadata(final StandardSy
     metadata.put("connection_id", standardSync.getConnectionId());
 
     final String frequencyString;
-    if (standardSync.getManual()) {
+    // TODO(https://github.com/airbytehq/airbyte/issues/2170): handle cron strings properly.
```

Comment:
So this basically relies on us "dual-writing" to the old `schedule` column, and the linked issue is for the clean-up task that will ultimately deprecate the old `schedule` column, right?

Would it make sense to update the code here right now to prefer the new column if set, and only fall back to `getSchedule` if the new column wasn't available? 

We could even add some logging that tells us whenever this code had to use the old schedule. That way, when we eventually get around to removing the old schedule, we'll have logs that can help improve our confidence that the column is indeed safe to remove